ui/redux/actions/reportContent.js

```diff
@@ -0,0 +1,61 @@
+// @flow
+import * as ACTIONS from 'constants/action_types';
+import { COPYRIGHT_ISSUES, OTHER_LEGAL_ISSUES } from 'constants/report_content';
+
+type Dispatch = (action: any) => any;
+
+export const doReportContent = (category: string, params: string) => (dispatch: Dispatch) => {
+  dispatch({
+    type: ACTIONS.REPORT_CONTENT_STARTED,
+  });
+
+  let RICK_URL;
```

Comment:
What does `RICK` mean?

Reply:
`REPORT_CONTENT_URL`

Reply:
nvm. This can probably be something like `REPORT_URL`